src/pugixml.hpp

```diff
@@ -1207,34 +1238,34 @@ namespace pugi
 		const xpath_parse_result& result() const;
 	};
 	#endif
-	
+
 	// XPath node class (either xml_node or xml_attribute)
 	class PUGIXML_CLASS xpath_node
 	{
 	private:
 		xml_node _node;
 		xml_attribute _attribute;
-	
+
 		typedef void (*unspecified_bool_type)(xpath_node***);
 
 	public:
 		// Default constructor; constructs empty XPath node
 		xpath_node();
-		
+
 		// Construct XPath node from XML node/attribute
 		xpath_node(const xml_node& node);
-		xpath_node(const xml_attribute& attribute, const xml_node& parent);
+		explicit xpath_node(const xml_attribute& attribute, const xml_node& parent);
```

Comment:
Given that xpath_node ctor with one argument is implicit, I don't see a reason for making this explicit. It can be conveniently used in C++11 occasionally e.g. for passing { attr, node }, and to keep the interface symmetric either both have to be implicit or explicit (but main ctor can't be made explicit because of compatibility - and, frankly, usability - so it does not make sense to make this one explicit).